### ISSUE TITLE
Support sending to SegWit/Bech32 addresses

### ISSUE THREAD

**Issue by cbeams** [bisq-network/projects#15]
## Description

Currently Bisq does not support sending funds from its wallet to external SegWit / Bech32 addresses. Being able to do so is amongst Bisq's most requested features. See detailed comment thread at https://github.com/bisq-network/bisq/issues/1139.

## Tasks

**Comment by cbeams** [comment]
I've labeled this as blocked on resources and am closing as deferred until that situation changes. See the comments at bisq-network/bisq#1139 for details.
